ci/install.bash

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+REPO_PATH=/workspace PROPERTIES_FILE=/workspace/properties CI=true /workspace/spinnaker-for-gcp/scripts/install/setup.sh
```

Comment:
Just a nit, but I expected `REPO_PATH` to be the full path to the repo, not the parent directory of the repo.

Reply:
good point, going to change to `PARENT_DIR`